01week/rockPaperScissors.js

```diff
@@ -13,6 +13,17 @@ const rl = readline.createInterface({
 
 // the function that will be called by the unit test below
 const rockPaperScissors = (hand1, hand2) => {
+  let Hand1 = hand1.toLowerCase().trim();
```

Comment:
I would have opted to use a different variable than just capitalizing the H in hand1/2